### PR TITLE
knight.py finished

### PR DESCRIPTION
**Pull request by lucas-mas**
update_possible_moves() method implemented using the code from move_utils.py (from commit 404258f1), with a few adjustments.

### CODE REVIEW

**Comment by lucas-mas** [comment]
I found a bug. Fixed

**Comment by luizlaljr** [comment]
I believe that the image of the piece must be instantiated in the specific child class.

File: models/knight.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    def __init__(self, row, col, color, img):
+        super(Knight, self).__init__(row, col, color, img)
+
+    def __str__(self):
+     return "Knight"
+
+    #-- Metodos auxiliares --#
+
+    # retorna os 2 L's: |--
+    def knight_left_L(self, pos):
```
Comment (yagoazedias) [inline models/knight.py]: [Opcional] Eu removeria o sufixo `L` dos nomes dos métodos, não acredito que tenha muito significado semântico.

Comment (lucas-mas) [reply models/knight.py]: Verdade. Removido

**Pull request merged**